sql/expression/div.go

```diff
@@ -717,9 +742,11 @@ func intDiv(ctx *sql.Context, lval, rval interface{}) (interface{}, error) {
 
 			// intDiv operation gets the integer part of the divided value
 			divRes := l.DivRound(r, 2)
```

Comment:
Could we pass in `0` here as the precision and avoid the call to `divRes.Truncate(0)` on line 748? Seems like that might be a little more direct, since we don't ever need those decimal digits.

Reply:
Ah DivRound rounds the result value with defined precision, but int division only takes int part without rounding the result, which is why I have it do this weird steps 😅

Reply:
I just made it more clear and added comment for this behavior.